cat/CaT-service.yaml

```diff
@@ -2566,17 +2568,88 @@ paths:
       security:
         - OAuth2:
             - buyer
-      summary: Request Supplier Further Info
+      summary: Request messages for an event
```

Comment:
As above - looks like a copy/paste from list response

Reply:
Made fix in latest version, thanks